src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.HashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "Cart")
```

Comment:
Jeśli tabela nazywa się tak samo jak encja na bazie to w ogóle nie dodawajmy adnotacji @Table. Ona nie jest do niczego wymagana